config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -234,7 +234,7 @@ periodics:
 #       # Azure-specific test args
 #       - --deployment=aksengine
 #       - --provider=skeleton
-#       - --aksengine-orchestratorRelease=1.18
+#       - --aksengine-orchestratorRelease=1.19
```

Comment:
I think we should remove this commented section all together. The changes in any config might not have made it to this and we should start from scratch if we want to add a new job.